test/sampleconsistency/sampleconsistency_test.go

```diff
@@ -85,8 +88,20 @@ func checkContains(t *testing.T, rl []string, src string) {
 		}
 		ir++
 	}
-	if is < len(sl) && best < len(sl) && best != -1 {
-		t.Fatalf("README.md file is missing line %d ('%s') from file '%s'", best, sl[best], src)
+
+	actionMsg := "All files required for running Helloworld sample apps are checked " +
```

Comment:
Why is this limited to HelloWorld? All the knative samples are required to have the synchronization between code and README.

Reply:
Removed `HelloWorld` from the error message, will let docs team triage the importance of other sample apps
@samodell